docs/sample_config.yaml

```diff
@@ -650,7 +650,7 @@ acme:
 # servers provided by user input.
 #
 # As of Synapse v1.24.0 this option also affects any outbound requests to push
-# servers provided by user input.
+# servers provided by user input and to key revocation requests.
```

Comment:
we should be clearer about what sort of "key" this is talking about. It might be clearer to say "for checking key validitity for third-party invite events".